packages/template-retail-react-app/app/components/cancel-order-modal/index.jsx

```diff
@@ -17,73 +17,146 @@ import {
     ModalBody,
     ModalCloseButton,
     Button,
-    Text
+    Menu,
+    MenuButton,
+    MenuList,
+    MenuItem,
+    Box,
+    Stack,
+    useBreakpointValue
 } from '@chakra-ui/react'
+import {ChevronDownIcon} from '@chakra-ui/icons'
+import OrderProducts from '@salesforce/retail-react-app/app/components/order-products'
+import {
+    messages,
+    CANCELLATION_REASONS
```

Comment:
I would suggest that you place `CANCELLATION_REASONS` in the `app/constants.js` file. Contrary to the file being named constants, this is typically a customization point that customers know to override this one file to redefine values that effect multiple parts of the app. Having this new constants file specific to this component will be less discoverable for developers and maybe a little more cumbersome.

Reply:
Gotcha. Makes sense and since this isn't going into develop I am ok with that.